@market/hooks/useOffchainOrders.ts

```diff
@@ -0,0 +1,19 @@
+import { fetchOffchainOrdersForToken } from 'services/offchainOrdersService'
+import useSWR from 'swr'
+
+import { NFTObject } from '@zoralabs/nft-hooks'
+
+export function useOffchainOrders(nft?: NFTObject) {
```

Comment:
I believe ideally the signature here should be `{ contractAddress, tokenId }`, so that no call would be dispatched initially when both values are `undefined`

Reply:
there's a chance `useSWR` keeps track of it, but having non-optional values just feels cleaner and more explicit to me